src/sentry/integrations/jira_server/integration.py

```diff
@@ -244,13 +286,188 @@ def get_client(self):
 
         return JiraServerClient(
             self.model.metadata["base_url"],
-            JiraServer(self.default_identity.data),
+            self.default_identity.data,
             self.model.metadata["verify_ssl"],
         )
 
+    def get_organization_config(self):
```

Comment:
Probably fine as is, but optionally you can convert all instances of `Jira` to `JiraServer` in case someone has both installed or something.

Might just be worth doing in the next PR though, upto you!